src/js_schema.hpp

```diff
@@ -87,6 +88,9 @@ static inline void parse_property_type(StringData object_name, Property& prop, S
     if (type == "bool") {
         prop.type |= PropertyType::Bool;
     }
+    else if (type == "mixed") {
```

Comment:
I believe `Mixed` is always optional (and you shouldn't explicitly set it). Should we add a check or leave it to object store?